web/src/components/tasks/TaxInput.tsx

```diff
@@ -106,6 +106,23 @@ export const TaxInput = (props: Props): ReactElement => {
     </div>
   );
 
+  const showNeedsAccountModalForGuest = (): (() => void) | undefined => {
+    if (isAuthenticated === IsAuthenticated.FALSE) {
+      return () => {
+        return setShowNeedsAccountModal(true);
+      };
+    }
+    return undefined;
+  };
```

Comment:
[dust] This is just nit picky. You don't need to explicitly return `undefined`. And you can also use the arrow function shorthand to make this a little smaller.
```suggestion
    if (isAuthenticated === IsAuthenticated.FALSE) {
      return () =>  setShowNeedsAccountModal(true);
    }
    return
  };
```

Reply:
for the undefined, I think I prefer the explicit undefined personally.